components/script/dom/workerglobalscope.rs

```diff
@@ -71,17 +71,17 @@ pub struct WorkerGlobalScope {
     #[ignore_heap_size_of = "Defined in std"]
     mem_profiler_chan: mem::ProfilerChan,
     #[ignore_heap_size_of = "Defined in ipc-channel"]
-    devtools_chan: Option<IpcSender<ScriptToDevtoolsControlMsg>>,
+    devtools_server_event_sender: Option<IpcSender<ScriptToDevtoolsControlMsg>>,
 
     #[ignore_heap_size_of = "Defined in ipc-channel"]
     /// Optional `IpcSender` for sending the `DevtoolScriptControlMsg`
     /// to the server from within the worker
-    devtools_sender: Option<IpcSender<DevtoolScriptControlMsg>>,
+    devtools_client_script_process_sender: Option<IpcSender<DevtoolScriptControlMsg>>,
 
     #[ignore_heap_size_of = "Defined in std"]
     /// This `Receiver` will be ignored later if the corresponding
     /// `IpcSender` doesn't exist
-    devtools_receiver: Receiver<DevtoolScriptControlMsg>,
+    devtools_client_script_process_receiver: Receiver<DevtoolScriptControlMsg>,
```

Comment:
Maybe using `to_devtools_sender`/`from_devtools_sender`/`from_devtools_receiver` would be better.